grasa_event_locator/models.py

```diff
@@ -6,7 +6,6 @@ class userInfo(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     org_name = models.CharField(max_length=255)
     isAdmin = models.BooleanField(default=False)
-    isActive = models.BooleanField(default=False)
```

Comment:
This will require a database schema change, right? Do we need to commit a migration file?